Extension/src/Utility/System/garbageCollector.ts

```diff
@@ -0,0 +1,14 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { setFlagsFromString } from 'v8';
+import { runInNewContext } from 'vm';
+
+setFlagsFromString('--expose_gc');
+const gc = runInNewContext('gc'); // nocommit

```

Comment:
This "nocommit" comment doesn't make any sense to me. Can you remove it or add/replace it with a comment that I can understand?

Reply:
Looks like a comment from the original place I found the code. removing